src/Input/TextField/TextField.tsx

```diff
@@ -87,15 +90,22 @@ const TextField: React.FunctionComponent<Props> = props => {
         aria-label={label}
         small={small}
         disableTyping={disableTyping}
+        hasStartIcon={hasStartIcon}
         {...restProps}
       />
+      {startIcon && (
+        <StartIconContainer className="start-icon">
```

Comment:
Is it possible to place `StartIconContainer` before `TextFieldInput` to avoid styling `left: 38px`  and `padding-left: 38px` for Input and Label?

Reply:
Even on keeping it before I have to add some margins. Also I have to add a z-index to icon to place it over the text field  else it is not visible. 🤔

Reply:
Ah I see, I thought the border is styled with `TextFieldContainer`. 😅 

Btw, I think we can make `38` be a const like `startIconPadding`.